tests/e2e/specs/api-cache.test.js

```diff
@@ -64,60 +72,28 @@ describe( 'API cache', () => {
 	} );
 
 	it( 'isolates client storage between sessions', async () => {
-		const firstTestNotification = { ...testSiteNotification };
-		const secondTestNotification = {
-			...testSiteNotification,
-			id: 'test-notification-2',
-			title: 'Test notification title 2',
-			dismissLabel: 'test dismiss site notification 2',
-		};
+		await goToWordPressDashboard();
 
-		// create first notification
-		await wpApiFetch( {
-			path: 'google-site-kit/v1/e2e/core/site/notifications',
-			method: 'post',
-			data: firstTestNotification,
+		const initialTimeData = await googleSiteKitAPIgetTime();
+		expect( initialTimeData ).toMatchObject( {
+			time: expect.any( Number ),
 		} );
 
-		await goToSiteKitDashboard();
-
-		await page.waitForSelector(
-			`#${ firstTestNotification.id }.googlesitekit-publisher-win--is-open`,
-			{ timeout: 10_000 } // Core site notifications are delayed 5s for surveys.
-		);
-
-		await expect( page ).toClick(
-			'.googlesitekit-publisher-win .mdc-button span',
-			{
-				text: firstTestNotification.dismissLabel,
-			}
-		);
-
-		// create second notification
-		await wpApiFetch( {
-			path: 'google-site-kit/v1/e2e/core/site/notifications',
-			method: 'post',
-			data: secondTestNotification,
-		} );
+		// Show that the data is cached when fetching again.
+		const timeData = await googleSiteKitAPIgetTime();
+		expect( timeData ).toEqual( initialTimeData );
 
 		// delete auth cookie to sign out the current user
```

Comment:
We might as well update this comment while we're here:

```suggestion
		// Delete auth cookie to sign out the current user.
```